### ISSUE TITLE
Google Fonts loads all subsets

### ISSUE THREAD

**Issue by mapsteps** [themeum/kirki#1871]
### Issue description:
Hey @aristath,

a client just pointed out that the webfontloader outputs all subsets - I assume that doesn't affect performance but wonder how it works. I couldn't find anything related on the internet so I thought I'd just ask you :) 

Here's an example:

```
<script type='text/javascript'>
WebFont.load({google:{families:['Open Sans:300:cyrillic,cyrillic-ext,devanagari,greek,greek-ext,khmer,latin,latin-ext,vietnamese,hebrew,arabic,bengali,gujarati,tamil,telugu,thai', 'Encode Sans:400:cyrillic,cyrillic-ext,devanagari,greek,greek-ext,khmer,latin,latin-ext,vietnamese,hebrew,arabic,bengali,gujarati,tamil,telugu,thai', 'Nobile:700:cyrillic,cyrillic-ext,devanagari,greek,greek-ext,khmer,latin,latin-ext,vietnamese,hebrew,arabic,bengali,gujarati,tamil,telugu,thai']}});
</script>
```

### Version used:
3.0.25

**Comment by aristath** [comment]
In previous versions googlefonts were loaded in the `<head>` of the document and they were render-blocking. That's why we used to have a separate subset control in typography fields allowing users to select the subset they wanted, so that we could minimize the requests.

However since we switched to the JS WebFont-Loader and webfonts are loaded asyncronously they don't impact performance. That allowed us to load all subsets and remove the separate dropdown from typography fields, while at the same time we keep compatibility for all languages since the js loader takes care of the order in which they get loaded :)
